src/components/SellerSearch.js

```diff
@@ -4,16 +4,29 @@ import styled from 'styled-components';
 // import { API } from '../config';
 
 const SellerSearch = props => {
-  const clickCheck = e => {
-    console.log('여기 찍힘');
-    console.log(e.target.className);
+  const [searchWord, setSearchWord] = useState([]);
+
+  const searchReady = keyword => {
+    setSearchWord(keyword);
   };
 
   return (
     <AllWrap>
       <SearchBox>
-        <SearchInput type="text" placeholder="셀러명을 검색해 주세요" />
-        <i className="fas fa-apple-alt fa-2x" onClick={e => clickCheck(e)}></i>
+        <SearchInput
+          type="text"
+          placeholder="셀러명을 검색해 주세요"
+          onChange={e => searchReady(e.target.value)}
```

Comment:
- 추가적인 로직이 들어가는 것이 아니라면, searchReady 라는 함수를 한번 더 생성해줄 필요가 없어 보입니다!
```suggestion
          onChange={e => setSearchWord(e.target.value)}
```